migrate/db-seed-commonjs-pkg/prisma/seed.ts

```diff
@@ -0,0 +1,71 @@
+import { PrismaClient, Prisma } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+const userData: Prisma.UserCreateInput[] = [
```

Comment:
Is this best practice how to define input for our create methods? 💡

Reply:
If you want to extract the object outside the prisma call, yes